Doc/library/doctest.rst

```diff
@@ -759,23 +759,27 @@ Warnings
 :mod:`doctest` is serious about requiring exact matches in expected output.  If
 even a single character doesn't match, the test fails.  This will probably
 surprise you a few times, as you learn exactly what Python does and doesn't
-guarantee about output.  For example, when printing a dict, Python doesn't
-guarantee that the key-value pairs will be printed in any particular order, so a
-test like ::
+guarantee about output.  For example, when printing a set, Python doesn't
+guarantee that the key is printed in any particular order, so a test like ::
```

Comment:
I would use elements in place of key, for members of a set.

Reply:
Concur with Julien.  Members of a set are "elements"

Reply:
Hi @JulienPalard and @rhettinger thank you for your catch,